apps/web/components/Skeletons/OrderAgainItem/OrderAgainItem.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div role="status" class="flex w-full mt-6 items-center animate-pulse">
+    <div
+      class="flex items-center justify-center w-24 h-24 flex-shrink-0 bg-gray-300 text-gray-600 rounded-sm mr-3 sm:w-40 sm:h-40"
+    >
+      <svg
```

Comment:
We should probably stick to our [default way](https://github.com/plentymarkets/plentyshop-pwa-docs/blob/main/guide/how-to/custom-icon.md)